src/Modules/LearningTasks/Tutor.LearningTasks.Tests/LearningTasksTestFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Tutor.BuildingBlocks.Tests;
+using Tutor.LearningTasks.Infrastructure.Database;
+
+namespace Tutor.LearningTasks.Tests;
+
+public class LearningTasksTestFactory : BaseTestFactory<LearningTasksContext>
+{
+    protected override IServiceCollection ReplaceNeededDbContexts(IServiceCollection services)
+    {
+        var descriptor = services.SingleOrDefault(d => d.ServiceType == typeof(DbContextOptions<LearningTasksContext>));
```

Comment:
Keep in mind that as you develop the LT module you will probably need to invoke "IsCourseOwner" for instructor use cases or "HasActiveEnrollment" for learner use cases. You will need the Courses context to test these use cases. Look to the KC module to see how these can be replaced.

Reply:
Ok, I will add Courses context following the approach used in the KC module.